vizro-core/src/vizro/actions/_action_loop/_build_action_loop_callbacks.py

```diff
@@ -22,12 +25,22 @@ def _build_action_loop_callbacks() -> None:
 
     gateway_inputs: List[Input] = []
     for actions_chain in actions_chains:
+        actions_chain_trigger_component_id = actions_chain.trigger.component_id
```

Comment:
> Is there any reason we have to assign this to a variable instead of using actions_chain.trigger.component_id directly? Does the dot notation cause issues?

The reason is that we modify or reuse this variable in three places below and it's nice to extract it once and then adjust and reuse it.

---

> Also, what do you think about renaming to actions_chain_trigger_model_id to be more generic?

The word "component" is used everywhere in the project when it refers to the trigger component_id (dash syntax). Also, this code is the best example of how action source (trigger) doesn't need to be Vizro model (but some pure dash underlying component).

Reply:
On your first point, got it - makes sense 👍 

Hmm.. yeah, I see where the confusion comes from. The term `components` is probably not the best - for me all models inside the projects `components` folder I consider as `components`, which is why I thought it would be better to rename to `actions_chain.trigger.model_id`, as we also have models outside of the `components` folder that can be triggers e.g. the `Page` model can be a trigger (at least it has an action argument attached), but is not a component for me in that sense.

I think your definition of components is more broad than mine as you consider all dash objects as components, whether they